Source/Forms/Gen4/Wild4.cpp

```diff
@@ -254,6 +261,42 @@ void Wild4::updatePokemonSearcher()
     }
 }
 
+void Wild4::updateTreesTypeGenerator()
+{
+    if (Encounters4::getHeadbuttSpecialFlag(currentProfile->getVersion(),
+                                            encounterGenerator[ui->comboBoxGeneratorLocation->currentData().toInt()].getLocation()))
+    {
+        if (ui->comboBoxGeneratorTreesType->findData(toInt(HeadbuttType::Special)) < 0)
```

Comment:
`< 0` -> `== -1`

According to the docs: `Returns the index of the item containing the given data for the given role; otherwise returns -1.`

Reply:
done